testbench/rest_server.py

```diff
@@ -889,6 +889,7 @@ def object_insert(bucket_name):
         db.insert_upload(upload)
         response = flask.make_response("")
         response.headers["Location"] = upload.location
+        response.headers["X-Goog-Upload-URL"] = upload.location
```

Comment:
Do you want to add a test to verify this works?

Reply:
Also, the documented header is `Location`:

https://cloud.google.com/storage/docs/performing-resumable-uploads

The `X-Goog-Upload-URL` seems to be undocumented, and undocumented behavior may change at any time.

Reply:
I'm a little confused about this. I tested out creating a resumable upload session and I actually don't see `x-goog-upload-url` in the response at all, only `location`.

However I do see it documented for other random Google APIs, e.g. https://developers.google.com/photos/library/guides/resumable-uploads#:~:text=The%20header%20field%20X%2DGoog%2DUpload%2DURL%20contains%20a,use%20it%20for%20subsequent%20requests.